commandline/WSL/troubleshooting.md

```diff
@@ -16,6 +16,26 @@ ms.assetid: 6753f1b2-200e-49cc-93a5-4323e1117246
   This is the first release of this brand new technology and it is branded “beta” deliberately – it’s not yet complete! We know there are issues and incomplete features, you should expect some things to work and for many things to fail. But we appreciate you playing with this feature and helping us identify the issues we need to fix in order to deliver a great experience.
 
 
+### Bash loses network connectivity once connected to a VPN
+
+If after connecting to a VPN on Windows, bash loses network connectivity, try this workaround from within bash. This workaround will allow you to manually override the DNS resolution through `/etc/resolv.conf`.
```

Comment:
Can we rephrase this to talk about Windows Subsystem for Linux instead of "bash"?